src/components/Garden/Preferences/EVMExecutor.js

```diff
@@ -100,29 +163,79 @@ function EVMExecutor() {
   const handleOnCreateIntent = useCallback(async () => {
     const description = radspec[actions.NEW_DECISION]()
     const type = actions.NEW_DECISION
-    const intent = await evmcrispr.encode(
-      [
-        evmcrispr
-          .call(installedApps[selectedApp])
-          [functionList[selectedFunction]](...parameters),
-      ],
-      ['disputable-voting'],
-      // TODO: just for now that for some reason the radspec description on the card is not working, after fixed we can ask the user for enter some forum post related to why the decision is being created
-      // { context: asciiToHex(functionList[selectedFunction]) }
-      // having some issue on the lib when passing the function that need to check with david
-      { context: 'new decision' }
-    )
+
+    let intent
+
+    if (interactionType === INTERNAL_INDEX) {
+      intent = await evmcrispr.encode(
+        [
+          evmcrispr
+            .call(installedApps[selectedApp])
+            [functionList[selectedFunction]](...parameters),
+        ],
+        ['disputable-voting'],
```

Comment:
Here we should generalize for cases like on `xdai` and `polygon` where the app we install is on the open AragonPM registry. So the `EVMCrispr` expect `disputable-voting.open` instead.

Reply:
nice cathc!